pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -246,10 +246,18 @@ type ProjectsVal struct {
 
 // defaultZones is the list of  zones used by default for cluster creation.
 // If the geo flag is specified, nodes are distributed between zones.
+// These are GCP zones available according to this page:
+// https://cloud.google.com/compute/docs/regions-zones#available
 var defaultZones = []string{
 	"us-east1-b",
 	"us-west1-b",
 	"europe-west2-b",
+	"us-east1-c",
```

Comment:
How about listing the zones in alphabetical order? Or would that affect the order in which nodes are chosen and thus make node placement for clusters with <= 6 nodes not be well distributed?

Reply:
When numNodes < numZones, `ZonePlacement` assigns the first `numNodes` (from `defaultZones`); otherwise, it assigns an equal number of zones (in the given order) modulo the remainder, if any. So, the order doesn't really matter except for highly unbalanced cases.